backend/FileSystem/FileManager.php

```diff
@@ -91,6 +91,9 @@ public function copyResource(
         }
     }
 
+    /**
+     * @var string[]
+     */
```

Comment:
It was for an array argument inside function, but ECS decided otherwise. 

It seems I need to put it inside function structure for it to function properly.